app/models/node.rb

```diff
@@ -681,7 +681,7 @@ def self.new_note(params)
                     comment: 2,
                     type:    'note')
     node.status = 4 if author.first_time_poster
-    node.draft if params[:draft] == "true"
+    node.save_draft if params[:draft] == "true"
 
```

Comment:
@Onyinye91-ctrl, I'm not sure why these changes were made, since the original issue didn't mention them? Can you revert these changes in another commit?